buildSrc/src/main/java/saros/gradle/intellij/IntellijConfigurator.java

```diff
@@ -12,6 +12,12 @@
 
   private static final String LOCKFILE_NAME = "saros_sandbox.lock";
   private static final String SANDBOX_DIR_PREFIX = "idea-sandbox-";
+  private static final String RACE_CONDITION_WARNING =
```

Comment:
What is the reason for all these newlines, especially inside the sentence?

Reply:
I removed the other linebreaks. The first is required to separate the warning from the exception message.